logic/src/commonMain/kotlin/com/wire/kalium/logic/data/prekey/PreyKeyMapper.kt

```diff
@@ -0,0 +1,9 @@
+package com.wire.kalium.logic.data.prekey
+
+import com.wire.kalium.network.api.prekey.PreKeyDTO
+
+class PreyKeyMapper {
```

Comment:
```suggestion
class PreKeyMapper {
```

Reply:
The existence of a `PreyKeyMapper` implies there's a `HunterKeyMapper` 😅